src/changebutton.js

```diff
@@ -0,0 +1,14 @@
+import { list } from './class.js';
+
+const changebutton = (button) => {
+  const icon = button.firstChild;
+  icon.setAttribute('name', 'trash-outline');
+  button.addEventListener('click', (e) => {
+    const item = e.target.parentNode;
+    console.log(item.parentNode.id);
+    list.remove(item.parentNode.id);
+    window.location.reload();
```

Comment:
- [ ] **Optional:** You could re-render your list using your display function rather than use `window.location.reload()`